library/Network/CloudSeeder/Main.hs

```diff
@@ -62,6 +68,10 @@ renderCliError (CliStackNotConfigured stackName)
 renderCliError (CliMissingDependencyStacks stackNames)
   =  "the following dependency stacks do not exist in AWS:\n"
   <> T.unlines (map ("  " <>) stackNames)
+renderCliError (CliTemplateDecodeFail decodeFailure)
+  = "template YAML decoding failed: " <> T.pack decodeFailure
+renderCliError CliTemplateMissingEnvParameter
```

Comment:
I wouldn’t treat `Env` specially; I think we should treat it identically to the other environment variables with the only exception that it’s required to generate the stack names.